travel-provider/TSP-30-RankTravelOptions/TSP-30-RankTravelOptions-Response.schema.json

```diff
@@ -27,13 +27,15 @@
         }
       },
       "required": [
-        "id", "dateTime", "engineVersion"
+        "id",
+        "dateTime",
+        "engineVersion"
       ],
       "additionalProperties": false
     },
-    "travelOptions": {
+    "rankedTravelOptions": {
       "type": "array",
-      "description": "A list of 1 to 9 travel options.",
+      "description": "A list of travel options ranked in order of best value first.",
```

Comment:
Can we remove the phrase 'best value first' from here - Just say 'A list of travel options ranked in order of preference from best first to worst last.'

Reply:
Never mind - fixed it myself